.vscode/tasks.json

```diff
@@ -6,7 +6,10 @@
         {
             "type": "npm",
             "script": "watch",
-            "problemMatcher": "$tsc-watch",
+            "problemMatcher": [
```

Comment:
I never quite understood this, what is the purpose of it?

Reply:
VS Code debug extension things.
It's ok to have it